From b8a78969f091e746776f640aee55c3e30769b3d3 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@gnome.org>
Date: Tue, 20 Oct 2026 18:31:56 -0500
Subject: [PATCH] Update ca-certificates

This update removes trust anchors that issued intermediate certificates
which have been compromised.

See: https://emudhra.we-have-a-problem.com/

Fixes #2090
---
 elements/components/ca-certificates.bst       |   6 +-
 .../remove-aaa-certificate-services.patch     | 161 ++++++++++++++++++
 .../remove-emsign-ecc-root-ca-g3.patch        | 128 ++++++++++++++
 3 files changed, 294 insertions(+), 1 deletion(-)
 create mode 100644 patches/ca-certificates/remove-aaa-certificate-services.patch
 create mode 100644 patches/ca-certificates/remove-emsign-ecc-root-ca-g3.patch

diff --git a/elements/components/ca-certificates.bst b/elements/components/ca-certificates.bst
index 17df718ea..baebced6f 100644
--- a/elements/components/ca-certificates.bst
+++ b/elements/components/ca-certificates.bst
@@ -198,7 +198,7 @@ sources:
 - kind: git_repo
   url: fedora_rpms:ca-certificates.git
   track: rawhide
-  ref: ca-certificates-2010_63-3_fc14-176-g725690846ee1ae1cd46e62c3a336c84820390d27
+  ref: ca-certificates-2010_63-3_fc14-179-g070c292b486904a007d155c8e2e4f484be29f790
 - kind: git_repo
   url: fedora_rpms:ca-certificates.git
   # Manually update: This is only used for update-ca-trust and its man
@@ -207,3 +207,7 @@ sources:
   # symlinks etc.
   ref: 91af9300e9ca630b72f466b317bc489446838db8
   directory: scripts
+- kind: patch
+  path: patches/ca-certificates/remove-aaa-certificate-services.patch
+- kind: patch
+  path: patches/ca-certificates/remove-emsign-ecc-root-ca-g3.patch
diff --git a/patches/ca-certificates/remove-aaa-certificate-services.patch b/patches/ca-certificates/remove-aaa-certificate-services.patch
new file mode 100644
index 000000000..76b0aced3
--- /dev/null
+++ b/patches/ca-certificates/remove-aaa-certificate-services.patch
@@ -0,0 +1,161 @@
+--- ../certdata.txt
++++ ../certdata.txt
+@@ -492,158 +492,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+ CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+ CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+ 
+-#
+-# Certificate "Comodo AAA Services root"
+-#
+-# Issuer: CN=AAA Certificate Services,O=Comodo CA Limited,L=Salford,ST=Greater Manchester,C=GB
+-# Serial Number: 1 (0x1)
+-# Subject: CN=AAA Certificate Services,O=Comodo CA Limited,L=Salford,ST=Greater Manchester,C=GB
+-# Not Valid Before: Thu Jan 01 00:00:00 2004
+-# Not Valid After : Sun Dec 31 23:59:59 2028
+-# Fingerprint (SHA-256): D7:A7:A0:FB:5D:7E:27:31:D7:71:E9:48:4E:BC:DE:F7:1D:5F:0C:3E:0A:29:48:78:2B:C8:3E:E0:EA:69:9E:F4
+-# Fingerprint (SHA1): D1:EB:23:A4:6D:17:D6:8F:D9:25:64:C2:F1:F1:60:17:64:D8:E3:49
+-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+-CKA_TOKEN CK_BBOOL CK_TRUE
+-CKA_PRIVATE CK_BBOOL CK_FALSE
+-CKA_MODIFIABLE CK_BBOOL CK_FALSE
+-CKA_LABEL UTF8 "Comodo AAA Services root"
+-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+-CKA_SUBJECT MULTILINE_OCTAL
+-\060\173\061\013\060\011\006\003\125\004\006\023\002\107\102\061
+-\033\060\031\006\003\125\004\010\014\022\107\162\145\141\164\145
+-\162\040\115\141\156\143\150\145\163\164\145\162\061\020\060\016
+-\006\003\125\004\007\014\007\123\141\154\146\157\162\144\061\032
+-\060\030\006\003\125\004\012\014\021\103\157\155\157\144\157\040
+-\103\101\040\114\151\155\151\164\145\144\061\041\060\037\006\003
+-\125\004\003\014\030\101\101\101\040\103\145\162\164\151\146\151
+-\143\141\164\145\040\123\145\162\166\151\143\145\163
+-END
+-CKA_ID UTF8 "0"
+-CKA_ISSUER MULTILINE_OCTAL
+-\060\173\061\013\060\011\006\003\125\004\006\023\002\107\102\061
+-\033\060\031\006\003\125\004\010\014\022\107\162\145\141\164\145
+-\162\040\115\141\156\143\150\145\163\164\145\162\061\020\060\016
+-\006\003\125\004\007\014\007\123\141\154\146\157\162\144\061\032
+-\060\030\006\003\125\004\012\014\021\103\157\155\157\144\157\040
+-\103\101\040\114\151\155\151\164\145\144\061\041\060\037\006\003
+-\125\004\003\014\030\101\101\101\040\103\145\162\164\151\146\151
+-\143\141\164\145\040\123\145\162\166\151\143\145\163
+-END
+-CKA_SERIAL_NUMBER MULTILINE_OCTAL
+-\002\001\001
+-END
+-CKA_VALUE MULTILINE_OCTAL
+-\060\202\004\062\060\202\003\032\240\003\002\001\002\002\001\001
+-\060\015\006\011\052\206\110\206\367\015\001\001\005\005\000\060
+-\173\061\013\060\011\006\003\125\004\006\023\002\107\102\061\033
+-\060\031\006\003\125\004\010\014\022\107\162\145\141\164\145\162
+-\040\115\141\156\143\150\145\163\164\145\162\061\020\060\016\006
+-\003\125\004\007\014\007\123\141\154\146\157\162\144\061\032\060
+-\030\006\003\125\004\012\014\021\103\157\155\157\144\157\040\103
+-\101\040\114\151\155\151\164\145\144\061\041\060\037\006\003\125
+-\004\003\014\030\101\101\101\040\103\145\162\164\151\146\151\143
+-\141\164\145\040\123\145\162\166\151\143\145\163\060\036\027\015
+-\060\064\060\061\060\061\060\060\060\060\060\060\132\027\015\062
+-\070\061\062\063\061\062\063\065\071\065\071\132\060\173\061\013
+-\060\011\006\003\125\004\006\023\002\107\102\061\033\060\031\006
+-\003\125\004\010\014\022\107\162\145\141\164\145\162\040\115\141
+-\156\143\150\145\163\164\145\162\061\020\060\016\006\003\125\004
+-\007\014\007\123\141\154\146\157\162\144\061\032\060\030\006\003
+-\125\004\012\014\021\103\157\155\157\144\157\040\103\101\040\114
+-\151\155\151\164\145\144\061\041\060\037\006\003\125\004\003\014
+-\030\101\101\101\040\103\145\162\164\151\146\151\143\141\164\145
+-\040\123\145\162\166\151\143\145\163\060\202\001\042\060\015\006
+-\011\052\206\110\206\367\015\001\001\001\005\000\003\202\001\017
+-\000\060\202\001\012\002\202\001\001\000\276\100\235\364\156\341
+-\352\166\207\034\115\105\104\216\276\106\310\203\006\235\301\052
+-\376\030\037\216\344\002\372\363\253\135\120\212\026\061\013\232
+-\006\320\305\160\042\315\111\055\124\143\314\266\156\150\106\013
+-\123\352\313\114\044\300\274\162\116\352\361\025\256\364\124\232
+-\022\012\303\172\262\063\140\342\332\211\125\363\042\130\363\336
+-\334\317\357\203\206\242\214\224\117\237\150\362\230\220\106\204
+-\047\307\166\277\343\314\065\054\213\136\007\144\145\202\300\110
+-\260\250\221\371\141\237\166\040\120\250\221\307\146\265\353\170
+-\142\003\126\360\212\032\023\352\061\243\036\240\231\375\070\366
+-\366\047\062\130\157\007\365\153\270\373\024\053\257\267\252\314
+-\326\143\137\163\214\332\005\231\250\070\250\313\027\170\066\121
+-\254\351\236\364\170\072\215\317\017\331\102\342\230\014\253\057
+-\237\016\001\336\357\237\231\111\361\055\337\254\164\115\033\230
+-\265\107\305\345\051\321\371\220\030\307\142\234\276\203\307\046
+-\173\076\212\045\307\300\335\235\346\065\150\020\040\235\217\330
+-\336\322\303\204\234\015\136\350\057\311\002\003\001\000\001\243
+-\201\300\060\201\275\060\035\006\003\125\035\016\004\026\004\024
+-\240\021\012\043\076\226\361\007\354\342\257\051\357\202\245\177
+-\320\060\244\264\060\016\006\003\125\035\017\001\001\377\004\004
+-\003\002\001\006\060\017\006\003\125\035\023\001\001\377\004\005
+-\060\003\001\001\377\060\173\006\003\125\035\037\004\164\060\162
+-\060\070\240\066\240\064\206\062\150\164\164\160\072\057\057\143
+-\162\154\056\143\157\155\157\144\157\143\141\056\143\157\155\057
+-\101\101\101\103\145\162\164\151\146\151\143\141\164\145\123\145
+-\162\166\151\143\145\163\056\143\162\154\060\066\240\064\240\062
+-\206\060\150\164\164\160\072\057\057\143\162\154\056\143\157\155
+-\157\144\157\056\156\145\164\057\101\101\101\103\145\162\164\151
+-\146\151\143\141\164\145\123\145\162\166\151\143\145\163\056\143
+-\162\154\060\015\006\011\052\206\110\206\367\015\001\001\005\005
+-\000\003\202\001\001\000\010\126\374\002\360\233\350\377\244\372
+-\326\173\306\104\200\316\117\304\305\366\000\130\314\246\266\274
+-\024\111\150\004\166\350\346\356\135\354\002\017\140\326\215\120
+-\030\117\046\116\001\343\346\260\245\356\277\274\164\124\101\277
+-\375\374\022\270\307\117\132\364\211\140\005\177\140\267\005\112
+-\363\366\361\302\277\304\271\164\206\266\055\175\153\314\322\363
+-\106\335\057\306\340\152\303\303\064\003\054\175\226\335\132\302
+-\016\247\012\231\301\005\213\253\014\057\363\134\072\317\154\067
+-\125\011\207\336\123\100\154\130\357\374\266\253\145\156\004\366
+-\033\334\074\340\132\025\306\236\331\361\131\110\060\041\145\003
+-\154\354\351\041\163\354\233\003\241\340\067\255\240\025\030\217
+-\372\272\002\316\247\054\251\020\023\054\324\345\010\046\253\042
+-\227\140\370\220\136\164\324\242\232\123\275\362\251\150\340\242
+-\156\302\327\154\261\243\017\236\277\353\150\347\126\362\256\362
+-\343\053\070\072\011\201\265\153\205\327\276\055\355\077\032\267
+-\262\143\342\365\142\054\202\324\152\000\101\120\361\071\203\237
+-\225\351\066\226\230\156
+-END
+-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
+-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+-
+-# Trust for "Comodo AAA Services root"
+-# Issuer: CN=AAA Certificate Services,O=Comodo CA Limited,L=Salford,ST=Greater Manchester,C=GB
+-# Serial Number: 1 (0x1)
+-# Subject: CN=AAA Certificate Services,O=Comodo CA Limited,L=Salford,ST=Greater Manchester,C=GB
+-# Not Valid Before: Thu Jan 01 00:00:00 2004
+-# Not Valid After : Sun Dec 31 23:59:59 2028
+-# Fingerprint (SHA-256): D7:A7:A0:FB:5D:7E:27:31:D7:71:E9:48:4E:BC:DE:F7:1D:5F:0C:3E:0A:29:48:78:2B:C8:3E:E0:EA:69:9E:F4
+-# Fingerprint (SHA1): D1:EB:23:A4:6D:17:D6:8F:D9:25:64:C2:F1:F1:60:17:64:D8:E3:49
+-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+-CKA_TOKEN CK_BBOOL CK_TRUE
+-CKA_PRIVATE CK_BBOOL CK_FALSE
+-CKA_MODIFIABLE CK_BBOOL CK_FALSE
+-CKA_LABEL UTF8 "Comodo AAA Services root"
+-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+-\321\353\043\244\155\027\326\217\331\045\144\302\361\361\140\027
+-\144\330\343\111
+-END
+-CKA_CERT_MD5_HASH MULTILINE_OCTAL
+-\111\171\004\260\353\207\031\254\107\260\274\021\121\233\164\320
+-END
+-CKA_ISSUER MULTILINE_OCTAL
+-\060\173\061\013\060\011\006\003\125\004\006\023\002\107\102\061
+-\033\060\031\006\003\125\004\010\014\022\107\162\145\141\164\145
+-\162\040\115\141\156\143\150\145\163\164\145\162\061\020\060\016
+-\006\003\125\004\007\014\007\123\141\154\146\157\162\144\061\032
+-\060\030\006\003\125\004\012\014\021\103\157\155\157\144\157\040
+-\103\101\040\114\151\155\151\164\145\144\061\041\060\037\006\003
+-\125\004\003\014\030\101\101\101\040\103\145\162\164\151\146\151
+-\143\141\164\145\040\123\145\162\166\151\143\145\163
+-END
+-CKA_SERIAL_NUMBER MULTILINE_OCTAL
+-\002\001\001
+-END
+-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+-
+ #
+ # Certificate "QuoVadis Root CA 2"
+ #
\ No newline at end of file
diff --git a/patches/ca-certificates/remove-emsign-ecc-root-ca-g3.patch b/patches/ca-certificates/remove-emsign-ecc-root-ca-g3.patch
new file mode 100644
index 000000000..5412f7b15
--- /dev/null
+++ b/patches/ca-certificates/remove-emsign-ecc-root-ca-g3.patch
@@ -0,0 +1,128 @@
+--- ../certdata.txt
++++ ../certdata.txt
+@@ -57851,125 +57851,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+ CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+ CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+ 
+-# Microsoft Code Signing Only Certificate
+-# Certificate "emSign ECC Root CA - G3"
+-# Issuer: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
+-# Serial Number:3c:f6:07:a9:68:70:0e:da:8b:84
+-# Subject: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
+-# Not Valid Before: Sun Feb 18 18:30:00 2018
+-# Not Valid After: Wed Feb 18 18:30:00 2043
+-# Fingerprint (MD5): ce:0b:72:d1:9f:88:8e:d0:50:03:e8:e3:b8:8b:67:40
+-# Fingerprint (SHA1): 30:43:fa:4f:f2:57:dc:a0:c3:80:ee:2e:58:ea:78:b2:3f:e6:bb:c1
+-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+-CKA_TOKEN CK_BBOOL CK_TRUE
+-CKA_PRIVATE CK_BBOOL CK_FALSE
+-CKA_MODIFIABLE CK_BBOOL CK_FALSE
+-CKA_LABEL UTF8 "emSign ECC Root CA - G3"
+-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+-CKA_SUBJECT MULTILINE_OCTAL
+-\060\153\061\013\060\011\006\003\125\004\006\023\002\111\116\061
+-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
+-\040\120\113\111\061\045\060\043\006\003\125\004\012\023\034\145
+-\115\165\144\150\162\141\040\124\145\143\150\156\157\154\157\147
+-\151\145\163\040\114\151\155\151\164\145\144\061\040\060\036\006
+-\003\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103
+-\040\122\157\157\164\040\103\101\040\055\040\107\063
+-END
+-CKA_ID UTF8 "0"
+-CKA_ISSUER MULTILINE_OCTAL
+-\060\153\061\013\060\011\006\003\125\004\006\023\002\111\116\061
+-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
+-\040\120\113\111\061\045\060\043\006\003\125\004\012\023\034\145
+-\115\165\144\150\162\141\040\124\145\143\150\156\157\154\157\147
+-\151\145\163\040\114\151\155\151\164\145\144\061\040\060\036\006
+-\003\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103
+-\040\122\157\157\164\040\103\101\040\055\040\107\063
+-END
+-CKA_SERIAL_NUMBER MULTILINE_OCTAL
+-\002\012\074\366\007\251\150\160\016\332\213\204
+-END
+-CKA_VALUE MULTILINE_OCTAL
+-\060\202\002\116\060\202\001\323\240\003\002\001\002\002\012\074
+-\366\007\251\150\160\016\332\213\204\060\012\006\010\052\206\110
+-\316\075\004\003\003\060\153\061\013\060\011\006\003\125\004\006
+-\023\002\111\116\061\023\060\021\006\003\125\004\013\023\012\145
+-\155\123\151\147\156\040\120\113\111\061\045\060\043\006\003\125
+-\004\012\023\034\145\115\165\144\150\162\141\040\124\145\143\150
+-\156\157\154\157\147\151\145\163\040\114\151\155\151\164\145\144
+-\061\040\060\036\006\003\125\004\003\023\027\145\155\123\151\147
+-\156\040\105\103\103\040\122\157\157\164\040\103\101\040\055\040
+-\107\063\060\036\027\015\061\070\060\062\061\070\061\070\063\060
+-\060\060\132\027\015\064\063\060\062\061\070\061\070\063\060\060
+-\060\132\060\153\061\013\060\011\006\003\125\004\006\023\002\111
+-\116\061\023\060\021\006\003\125\004\013\023\012\145\155\123\151
+-\147\156\040\120\113\111\061\045\060\043\006\003\125\004\012\023
+-\034\145\115\165\144\150\162\141\040\124\145\143\150\156\157\154
+-\157\147\151\145\163\040\114\151\155\151\164\145\144\061\040\060
+-\036\006\003\125\004\003\023\027\145\155\123\151\147\156\040\105
+-\103\103\040\122\157\157\164\040\103\101\040\055\040\107\063\060
+-\166\060\020\006\007\052\206\110\316\075\002\001\006\005\053\201
+-\004\000\042\003\142\000\004\043\245\014\270\055\022\365\050\363
+-\261\262\335\342\002\022\200\236\071\137\111\115\237\311\045\064
+-\131\164\354\273\006\034\347\300\162\257\350\256\057\341\101\124
+-\207\024\250\112\262\350\174\202\346\133\152\265\334\263\165\316
+-\213\006\320\206\043\277\106\325\216\017\077\004\364\327\034\222
+-\176\366\245\143\302\365\137\216\056\117\241\030\031\002\053\062
+-\012\202\144\175\026\223\321\243\102\060\100\060\035\006\003\125
+-\035\016\004\026\004\024\174\135\002\204\023\324\314\212\233\201
+-\316\027\034\056\051\036\234\110\143\102\060\016\006\003\125\035
+-\017\001\001\377\004\004\003\002\001\006\060\017\006\003\125\035
+-\023\001\001\377\004\005\060\003\001\001\377\060\012\006\010\052
+-\206\110\316\075\004\003\003\003\151\000\060\146\002\061\000\276
+-\363\141\317\002\020\035\144\225\007\270\030\156\210\205\005\057
+-\203\010\027\220\312\037\212\114\350\015\033\172\261\255\325\201
+-\011\107\357\073\254\010\004\174\134\231\261\355\107\007\322\002
+-\061\000\235\272\125\374\251\112\350\355\355\346\166\001\102\173
+-\310\370\140\331\215\121\213\125\073\373\214\173\353\145\011\303
+-\370\226\315\107\250\202\362\026\125\167\044\176\022\020\225\004
+-\054\243
+-END
+-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_FALSE
+-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+-
+-# Microsoft Code Signing Only Certificate
+-# Trust for "emSign ECC Root CA - G3"
+-# Issuer: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
+-# Serial Number:3c:f6:07:a9:68:70:0e:da:8b:84
+-# Subject: CN=emSign ECC Root CA - G3,O=eMudhra Technologies Limited,OU=emSign PKI,C=IN
+-# Not Valid Before: Sun Feb 18 18:30:00 2018
+-# Not Valid After: Wed Feb 18 18:30:00 2043
+-# Fingerprint (MD5): ce:0b:72:d1:9f:88:8e:d0:50:03:e8:e3:b8:8b:67:40
+-# Fingerprint (SHA1): 30:43:fa:4f:f2:57:dc:a0:c3:80:ee:2e:58:ea:78:b2:3f:e6:bb:c1
+-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+-CKA_TOKEN CK_BBOOL CK_TRUE
+-CKA_PRIVATE CK_BBOOL CK_FALSE
+-CKA_MODIFIABLE CK_BBOOL CK_FALSE
+-CKA_LABEL UTF8 "emSign ECC Root CA - G3"
+-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+-\060\103\372\117\362\127\334\240\303\200\356\056\130\352\170\262
+-\077\346\273\301
+-END
+-CKA_CERT_MD5_HASH MULTILINE_OCTAL
+-\316\013\162\321\237\210\216\320\120\003\350\343\270\213\147\100
+-END
+-CKA_ISSUER MULTILINE_OCTAL
+-\060\153\061\013\060\011\006\003\125\004\006\023\002\111\116\061
+-\023\060\021\006\003\125\004\013\023\012\145\155\123\151\147\156
+-\040\120\113\111\061\045\060\043\006\003\125\004\012\023\034\145
+-\115\165\144\150\162\141\040\124\145\143\150\156\157\154\157\147
+-\151\145\163\040\114\151\155\151\164\145\144\061\040\060\036\006
+-\003\125\004\003\023\027\145\155\123\151\147\156\040\105\103\103
+-\040\122\157\157\164\040\103\101\040\055\040\107\063
+-END
+-CKA_SERIAL_NUMBER MULTILINE_OCTAL
+-\002\012\074\366\007\251\150\160\016\332\213\204
+-END
+-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+-
+ # Microsoft Code Signing Only Certificate
+ # Certificate "emSign Root CA - C2"
+ # Issuer: CN=emSign Root CA - C2,O=eMudhra Inc,OU=emSign PKI,C=US
\ No newline at end of file
-- 
2.55.0

